docs/custom-registries.md

```diff
@@ -526,6 +526,19 @@ https://static.rust-lang.org/dist/rust-1.75.0-aarch64-unknown-linux-gnu.tar.gz
 https://static.rust-lang.org/dist/rust-1.75.0-x86_64-unknown-linux-gnu.tar.gz
 ```
 
+## `rye`
+
+Rye releases are downloaded from:
+
+- `https://github.com/astral-sh/rye/releases`
+
+Samples:
+
+```txt
+https://github.com/astral-sh/rye/releases/download/0.34.0/rye-x86_64-linux.gz.sha256
+https://github.com/astral-sh/rye/releases/download/0.34.0/rye-x86_64-linux.gz
```

Comment:
please add an arm64 sample

Reply:
@vhdirk I already looked them up for you to save you a few minutes ;)

`https://github.com/astral-sh/rye/releases/download/0.34.0/rye-aarch64-linux.gz`
`https://github.com/astral-sh/rye/releases/download/0.34.0/rye-x86_64-linux.gz.sha256`

Reply:
please add them to the docs